lib/config/stylistic.js

```diff
@@ -3,7 +3,11 @@
 module.exports = {
   rules: {
     'block-indentation': true,
+    'eol-last': 'always',
```

Comment:
I think we should change this to honor the `.editorconfig`. As it stands, enabling this (via `always`) means we error for `ember g component foo` because the default ember-cli `.editorconfig` states that we should **not** have an EOL in `.hbs` files ([see here](https://github.com/ember-cli/ember-new-output/blob/v3.15.1/.editorconfig#L16-L17)).

I'd say we update this to use the editor config setup similar to #918 and #919. That way, whatever your `.editorconfig` value has will be used and essentially make `'eol-last': true` automatically use either `always` or `never` appropriately.

Reply:
Agreed with that. I removed this rule from the config. We can consider adding it once it has been updated to read from `.editorconfig`.

Reply:
Submitted https://github.com/ember-template-lint/ember-template-lint/pull/1062 to add that support.

Reply:
@bmish - If you rebase, you can bring this back now (with the `'editorconfig'` setting added in #1062).

Reply:
Added it back. Thanks!